RecoVertex/AdaptiveVertexFinder/interface/TrackVertexArbitration.h

```diff
@@ -156,10 +170,13 @@ std::vector<VTX> TrackVertexArbitration<VTX>::trackVertexArbitrator(
         VertexDistance3D vdist;
         GlobalPoint ppv(pv.position().x(),pv.position().y(),pv.position().z());
 
-        std::map<unsigned int, Measurement1D> cachedIP;  
+        std::unordered_map<unsigned int, Measurement1D> cachedIP;  
         for(typename std::vector<VTX>::const_iterator sv = secondaryVertices.begin();
 	    sv != secondaryVertices.end(); ++sv) {
 
+	  const bool svHasTime = ( svhelper::cov33(*sv) > 0. );
```

Comment:
maybe swap the order of this line and the next line so this line can use `svTimeCov` instead of calling `cov33()` again? (probably very minor)

Reply:
I should just inline the cov33() function, instead.

Reply:
Likely done by the compiler already.